### DISCUSSION TITLE
[FR] min/max value to slider

### DISCUSSION THREAD

**Discussion by radhoo2k10** [Clooos/Bubble-Card#304] [Feature requests]
<html>
<body>
<p dir="auto" style="box-sizing: border-box; margin-top: 0px !important; margin-bottom: 16px; color: rgb(230, 237, 243); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, &quot;Noto Sans&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(13, 17, 23); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">Is it possible to add min and max value to the slider?<br style="box-sizing: border-box;">Like the My Slider V2 card has.</p>

min | number | 0 | Minimum value you can set the entity state
-- | -- | -- | --
max | number | 100 | Maximum value you can set the entity state

<span style="color: rgb(230, 237, 243); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, &quot;Noto Sans&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(13, 17, 23); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial; display: inline !important; float: none;">This way I can control the brightness so that my daughter don't blast the house with lights all the time :)</span>
</body>
</html>

**Comment by Inrego** [comment]
I would also like this. Basically to set a non-zero min value, so I can more easily dim the lights to 1%

**Comment by exetico** [comment]
Bump 😄 

_I'm trying to move some of my things to Bubble Cards, after v2 have dropped. It's awesome that quite a few new things are allowed. So, that's nice! I've tried and search through the README, but I can't find "limit" / "interval" / "max value", which I'd really like._

<img width="563" alt="image" src="https://github.com/Clooos/Bubble-Card/assets/3549445/f45ea778-20ac-4b70-aabc-f013e29b22da">

My Sonos speaks will blast the windows on 100% (no, not really, but, crazy load). I'd be nice to have an option to limit the range, maybe so it's just limiting it to 80% in the render if 100% slided.

Or, alternative, show that it's a limited scope, and let the slider from left to right reflect the 0-80% range.

I know it's quite specific, but for volume control it would me pretty awesome to have.

**Reply by Clooos** [reply]
Indeed! It was something I wanted to add in the v2, but there was so many new features that I had to stop somewhere 😅

But I will add this probably soon! This will be not too difficult I guess.

**Reply by exetico** [reply]
Thats promising news, and thank you for your initial thoughts! Fingers crossed 🤞.

**Reply by zimbo86** [reply]
i second this, got multiple use cases for this..

**Reply by hanseandersson** [reply]
> Indeed! It was something I wanted to add in the v2, but there was so many new features that I had to stop somewhere 😅
> 
> But I will add this probably soon! This will be not too difficult I guess.

Was this added to last update? It's the only thing holding me back from going all in och bubble cards :)

**Reply by kubayz** [reply]
I would also appreciate this feature for the media_player card. I do not use a volume higher than volume_level: 0.20 on the TV, so it is practically impossible to specify a specific value on the long slider.

**Reply by klaptafel** [reply]
Just looking for this as well! 😄

**Reply by Inrego** [reply]
@Clooos any updates on this?

**Reply by Clooos** [reply]
This will be finally a feature in the next beta! 😃

**Reply by exetico** [reply]
What a wonderful information! I'm so happy to hear that.
